### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Server_Authorization_Keyboard: At edit and cancel buttons, when we are pressing enter focus is going to the starting of the page

### ISSUE THREAD

**Issue by InduPriya1805** [Microsoft/IIS.WebManager#121]
GitHubTags: #A11y_Manage.IIS.Net; #A11yMAS; #A11yTCS; #C+E-H; #DesktopWeb; #Win10; #.NETWebsites; #Edge; #MAS2.4.3; #Keyboard;

**Environment Details:**
URL: https://manage.iis.net/webserver/authorization
Browser Details: Microsoft Edge 42.17134.1.0
MAS ID Violated: MAS 2.4.3

**Repro Steps:**
1. Download and install "Microsoft IIS Administration" application.
2. Enable Internet information Services.
3. Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4. Tab till Web Server and press enter
5. Tab till Authorization and press enter
6. Tab till more button (...) and press enter
7. Tab till edit button and press enter
8. Verify on pressing enter at edit button focus is going to starting of the page

**Actual Result:**
On pressing enter at edit button focus is going to the starting of the page

**Expected Result:**
On pressing enter at edit button focus should not go the starting of the page

**Note:**
Similarly on pressing enter at cancel button, focus is going to starting of the page
This issue is observed in all the pages in the application which has similar controls

**User Impact:**
Users with disabilities finds its difficult in working through the application if focus goes to the starting of the page

**Attachments:**
[Bug121_web server_authorization_focus going to starting of the page.pptx](https://github.com/Microsoft/IIS.WebManager/files/2326849/Bug121_web.server_authorization_focus.going.to.starting.of.the.page.pptx)

**Comment by jhkimnew** [comment]
Fixed

**Comment by Yashi1201** [comment]
**Github tags:** #A11ySev2;

**Comment by sspatil123** [comment]
#closed;
manage.iis.net is deprecated. The "Manage IIS" button on the www.iis.net home page will  be removed.
